README.md

```diff
@@ -2,8 +2,39 @@
 Use Adyen's plugin for PrestaShop to offer frictionless payments online, in-app, and in-store.
 
 ## Integration
-The plugin integrates card component(Secured Fields) using Adyen Checkout for all card payments. Local payment methods are integrated with Checkout Payment Methods.
-
+The plugin integrates card component (Secured Fields) using Adyen Checkout for all card payments.
```

Comment:
I think this can stay: "Local payment methods are integrated with Checkout Payment Methods."

Reply:
https://github.com/Adyen/adyen-prestashop/pull/204/commits/93ed4c3d149d833b3312d51674d4de8d52816a81